packages/sysutils/usbutils/package.mk

```
@@ -1,14 +1,22 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 # Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
+# Copyright (C) 2020-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="usbutils"
-PKG_VERSION="008"
-PKG_SHA256="44741af0bae9d402a0ef160a29b2fa700bb656ab5e0a4b3343d51249c2a44c8c"
+PKG_VERSION="013"
+PKG_SHA256="9e23494fcc78b7a80ee29a07dd179c95ae2f71509c35728dbbabc2d1cca41338"
 PKG_LICENSE="GPL"
 PKG_SITE="http://www.linux-usb.org/"
 PKG_URL="http://kernel.org/pub/linux/utils/usb/usbutils/$PKG_NAME-$PKG_VERSION.tar.xz"
 PKG_DEPENDS_TARGET="toolchain libusb systemd"
 PKG_LONGDESC="This package contains various utilities for inspecting and setting of devices connected to the USB bus."
+PKG_TOOLCHAIN="configure"
+
+pre_configure_target() {
+  (
+    sh autogen.sh
+  )
```

```suggestion
    ./autogen.sh
```

i think ? not buildtested but we already do it at other packages too

---

I have just now build tested it